sphinxcontrib/images.py

```diff
@@ -346,7 +348,8 @@ def setup(app):
     global DEFAULT_CONFIG
     app.require_sphinx('1.0')
     app.add_config_value('images_config', DEFAULT_CONFIG, 'env')
```

Comment:
Since this isn't actually needed because the configuration will be merged anyways, it might make sense to just pass a dict which also wouldn't break when copying it.

```suggestion
    app.add_config_value('images_config', {}, 'env')
```